src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java

```diff
@@ -178,7 +187,7 @@ private SystemState defaultStateTransfer()
                 newState = SystemState.EJECTING;
                 break;
             case BRING_BALL_TO_TOP:
-                if (mSystemState != SystemState.RAMPING || mSystemState != SystemState.HOLDING)
+                if (mSystemState != SystemState.RAMPING || mSystemState != SystemState.HOLDING) // FIXME: what's this do?
```

Comment:
We're just making sure that we don't transition to the first state in this state chain if we're already in the state chain. The `mSystemState != SystemState.HOLDING` is actually redundant right now, but we're leaving it in in case someone makes a change (then they don't have to think about this or try to stop the bug.)